### ISSUE TITLE
Can't rename page to name of (just) deleted other page

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-infrastructure/roswiki#274]
Trying to rename [wiki/universal_robot](http://wiki.ros.org/universal_robot) to `universal_robots` (plural) seems to fail as there is already a page with that name (I know, I created it).

After deleting the `universal_robots` page, tried renaming `universal_robot` again, but the same error is returned by the wiki:

> A page with the name 'universal_robots' already exists. Try a different name.

Is there some server-local caching issue?

---

Apologies if this is not the place to post issues with (use of) the wiki. I couldn't choose between ROS Answers or this tracker.

**Comment by tfoote** [comment]
I successfully moved it. The problem is that Delete is not really a delete in moin moin. It just revises the pages to have zero content. It still had the page history. I rolled the page back to the old content and then moved it to http://wiki.ros.org/universal_robots_old_redirect Then I was able to make the move.

We do periodically clear deleted pages from the backend to avoid clutter. So it would have worked eventually ~ 1 week.
